operator/src/main/resources/scripts/utils_base.sh

```diff
@@ -234,6 +234,17 @@ function initAuxiliaryImage() {
     return
   fi
 
+  if [[ "$AUXILIARY_IMAGE_COMMAND" == "cp -R \$AUXILIARY_IMAGE_PATH/* \$AUXILIARY_IMAGE_TARGET_PATH" ]]; then
```

Comment:
This looks to me like a somewhat fragile way to check if COMMAND is at the default. Any small change in the default at the operator level would forever more skip this check.

Instead:

-Could the operator pass some sort of flag that the COMMAND is at the default? 
 
-Or could you add a unit test in the operator that checks the default for COMMAND exactly matches this string, and have a notation in the unit test that the unit test exists because there's a corresponding equals check in `utils_base.sh`.

Something like that.

Reply:
- I thought about introducing another environment variable for the default command. However, adding env variable will cause the pods to roll since the pod specs will change due to that. 
- I will check and add the unit test if one doesn't exist already.  Thanks.